### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - Product Video] - Caption does not define for video player under 'Product Video'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#46]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.6" control and select it.
5. Navigate to 'Scenario' control and activate it.
6. Navigate to 'Product Video.Json' and activate it.
7. Card will appear.
8. Navigate to video and select 'Play button. Navigate all the video controls.
9. Verify the issue. 

**Actual Result:**
Caption does not define for video.

**Expected Result:**
Caption should be defined for pre-recorded video.

**User Experience:**
If Caption does not define on video. so, it would be difficult for users to know about video and purpose of video.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/captions-prerecorded

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/06c24afd-bdb2-49d0-95ee-fde32aea010e

**Comment by vagpt** [comment]
Hi @ardlank,

Hope you are doing great.

Could you please provide us ETA to fix this issue as due to this issue product is getting non-compliant.

**Comment by hggz** [comment]
Hey @vagpt, apologies for the delay on the update. We had to do some more investigation on the background for the issue and the existing behavior.

This actually isn't a bug right now. The native controls support captions however the video source needs to have the captions encoded in it. At that point the native video controls will update to support the caption option in the area you showed.

The video provided in the adaptive card test case **does not** have the captions encoded, and instead provides them in a different file. Processing separate caption files from the video source will be considered a feature enhancement.

We'll shift this to a separate feature to provide support for parsing and leveraging provided caption files with videos on iOS.

**Comment by VikrantSingh01** [comment]
Discussed with Arvind Prakash and Ajay Sharma. Will be treated as feature item as is part of 3454571 work item.
